docs/configure

```diff
@@ -492,7 +487,6 @@ dnslookup:
   domains = ! +local_domains
   transport = remote_smtp
   ignore_target_hosts = 0.0.0.0 : 127.0.0.0/8
-  headers_remove = X-Spam-Report:X-Spam-Score
```

Comment:
> For this reason, I currently use this header to set the filter:
> `X-Spam-Report: Spam detection software, running on the system "example.org",`
> the `X-Spam-Report`-tag in combination with `example.org` is unique for my mail server. I am not happy with that at all.

Why's this bad?
Regarding other things, I don't think I have enough knowledge to talk. After I migrated my setup to 2.3 couple days ago, my spam filtering became worse. And I don't even know how to start fixing that...